static/docs/commands-reference/status.md

```diff
@@ -53,19 +53,40 @@ This says that no differences were detected, and therefore that no stages would
 be run again if `dvc repro` were executed.
 
 If instead, differences are detected, `dvc status` lists those changes. For each
-DVC-file (stage) with differences, the _dependencies_ and/or _outputs_ that
-differ are listed. For each item listed, either the file name or the checksum is
-shown, and additionally a status word is shown describing the change:
+DVC-file (stage) with differences, the changes in both _dependencies_ and/or
+_outputs_ that differ are listed. For each item listed, either the file name or
+the checksum is shown, and additionally, a status word is shown describing the
+changes in checksum or status for both local and cloud workspace. Given list
+provides a reference to both, the status of the DVC-file and differences in the
+DVC-files:
 
 - For the local workspace:
-  - _changed_ means the file has changed
+
+  - _changed checksum_ means actual checksum of the DVC-file doesn't match the
+    one specified in it in DVC-files
+  - _always changed_ means that this is a special DVC-file with no dependencies,
+    which is considered always changed
+  - _changed deps_ means that there are some changes in dependencies that are
+    incorporated in the DVC-file, some of the prominent states are:
+    - _new_ : when new dependencies are added into DVC-file
```

Comment:
in this and the following items: it's more about the state of a file or a directory that is dependency, not about changes in the DVC-file. Let's discuss this on Discrord.